GX Password/strings/nl/Resources.resw

```diff
@@ -0,0 +1,162 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="DescApp" xml:space="preserve">
+    <value>Een Snelle, Makkelijke en Veilige Wachtwoordgenerator</value>
+  </data>
+  <data name="passLeng.Text" xml:space="preserve">
+    <value>Wachtwoord Lengte:</value>
+  </data>
+  <data name="passOpt.Text" xml:space="preserve">
+    <value>Wachtwoord Opties:</value>
+  </data>
+  <data name="sBtClipboard.Content" xml:space="preserve">
+    <value>Klembord</value>
+  </data>
+  <data name="sBtOk.Content" xml:space="preserve">
+    <value>Genereer Wachtwoord</value>
+  </data>
+  <data name="sTsLett.OffContent" xml:space="preserve">
+    <value>Letters Uitgeschakeld</value>
+  </data>
+  <data name="sTsLett.OnContent" xml:space="preserve">
+    <value>Letters Ingeschakeld</value>
+  </data>
+  <data name="sTsNumb.OffContent" xml:space="preserve">
+    <value>Nummers Uitgeschakeld</value>
+  </data>
+  <data name="sTsNumb.OnContent" xml:space="preserve">
+    <value>Nummers Ingeschakeld</value>
+  </data>
+  <data name="sTsSimi.OffContent" xml:space="preserve">
+    <value>Vergelijkbare Karakters Uitgeschakeld</value>
+  </data>
+  <data name="sTsSimi.OnContent" xml:space="preserve">
+    <value>Vergelijkbare Karakters Ingeschakeld</value>
+  </data>
+  <data name="sTsSymb.OffContent" xml:space="preserve">
+    <value>Symbolen Uitgeschakeld</value>
+  </data>
+  <data name="sTsSymb.OnContent" xml:space="preserve">
+    <value>DSymbolen Uitgeschakeld</value>
```

Comment:
`DSymbolen` Is that correct?

Reply:
Yes that is correct.

Reply:
I notice now however that I used disabled twice
Oh and the D sorry. I wasn't paying enough attention.